GLMakie/src/new-primitives.jl

```diff
@@ -314,11 +314,16 @@ function draw_atomic(screen::Screen, scene::Scene, plot::Scatter)
     # We register the screen under a unique name. If the screen closes
     # Any computation that depens on screen gets removed
     atlas = gl_texture_atlas()
-    add_input!(attr, :atlas, atlas)
+    if haskey(attr, :atlas)
```

Comment:
we'll need to do something like `:atlas_64_2048` for disambiguation with WGLMakie.
I already added this here:
https://github.com/MakieOrg/Makie.jl/blob/sd/compute-graph/src/utilities/texture_atlas.jl#L542
I guess we should normalize the adding of the texture atlas somehow... Maybe already in the the Plot constructor.
But I guess this doesn't need to happen in this PR.